store/sqlstore/channel_store.go

```diff
@@ -2465,48 +2570,84 @@ func (s SqlChannelStore) GetMembersForUserWithPagination(teamId, userId string,
 	return dbMembers.ToModel(), nil
 }
 
-func (s SqlChannelStore) AutocompleteInTeam(teamId string, term string, includeDeleted bool) (model.ChannelList, error) {
-	deleteFilter := "AND Channels.DeleteAt = 0"
-	if includeDeleted {
-		deleteFilter = ""
+func (s SqlChannelStore) GetTeamMembersForChannel(channelID string) ([]string, error) {
+	teamMemberIDs := []string{}
+	if err := s.GetReplicaX().Select(&teamMemberIDs, `SELECT tm.UserId
```

Comment:
Do we mean using `GetReplicaX()`?

Reply:
Didn't get this. Can you clarify?

Reply:
Did you mean calling the sqlx method instead of the gorp one?

Reply:
Yes, I meant the sqlx method.